docs/install/plugins.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Standalone Plugin Install
+parent: Install and Configure
+nav_order: 98
+---
+
+# Standalone plugin install
+
+If you don't want to use the all-in-one Open Distro for Elasticsearch Docker images or RPM packages, you can install the Security, Alerting, SQL, and Performance Analyzer plugins on a compatible Elasticsearch cluster just like any other Elasticsearch plugin. Navigate to the Elasticsearch home directory (likely `/usr/share/elasticsearch`) and run:
```

Comment:
What about kibana? If someone install security plugin, I think security kibana is also needed?

Reply:
Do we have the Kibana plugins available, too? I'm happy to add them, but the CloudFront URL is locked down such that you can't just browse for all available files.